packages/scene-composer/src/components/panels/scene-components/common/ValueDataBindingBuilder.tsx

```diff
@@ -193,14 +204,53 @@ export const ValueDataBindingBuilder: React.FC<IValueDataBindingBuilderProps> =
               }
               key={definition.fieldName}
             >
-              <Select
-                data-testid='value-data-binding-builder-select'
-                selectedOption={selectedOption}
-                onChange={onSelectChange(definition.fieldName, index)}
-                options={options}
-                disabled={disabled}
-                placeholder={intl.formatMessage({ defaultMessage: 'Select an option', description: 'placeholder' })}
-              />
+              {definition.fieldName === 'propertyName' && (
+                <div className='copy-select-flex'>
+                  <Box>
+                    {selectedOption && (
+                      <Popover
+                        size='small'
+                        position='top'
+                        triggerType='custom'
+                        dismissButton={false}
+                        content={<StatusIndicator type='success'>{selectedOption.label + ' copied'} </StatusIndicator>}
+                      >
+                        <Button
+                          variant='icon'
+                          iconName='copy'
+                          ariaLabel='Copy property name'
+                          onClick={() => {
+                            navigator.clipboard.writeText(selectedOption!.label ? selectedOption!.label : '');
```

Comment:
Return value of `window.clipboard.writeText` is a promise. so you need to cover them with async and await.